src/coreclr/jit/gentree.cpp

```diff
@@ -19995,7 +19995,7 @@ bool GenTree::SupportsSettingZeroFlag()
     }
 #endif
 #elif defined(TARGET_ARM64)
-    if (OperIs(GT_AND))
+    if (OperIs(GT_AND, GT_AND_NOT))
```

Comment:
any reason why this change was needed? What if we don't do the `BIC(s)` optimization? Will this still hold true?